### PR TITLE
Add About Landing page

### PR DESCRIPTION
**Pull request by pollygee**
@meg-gutshall @seanmarcia I've done as much as I know how to do on this page.  
I don't know how to get the link to the anchor work from the exclusivity and equity section.
I don't know how to make the background change color all the way across the page over the sections that don't span the whole page and I don't know the color for that background stripe.

I don't know if it's helpful, but I can try to keep building out as much as I know how to on some pages and have someone else do some of the styling?  If that's not helpful, let me know what would be helpful.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -4,6 +4,7 @@
   Version: 1.0
 */
```
Comment (pollygee) [inline assets/css/style.css:5]: I'm not sure why there are so many changes to this file.  I rebased from master and there were big changes in master.  I'm not sure why they're showing up as a diff from master to my branch though.  🤔

Comment (meg-gutshall) [reply assets/css/style.css:5]: Do you have your tab set to 4 spaces? I have mine set to 2. It looks like that could be what's causing this.

Comment (meg-gutshall) [reply assets/css/style.css:5]: Also there's no newline spacing between CSS rules on the master branch and there is on this one. Do you have a formatting plugin or something maybe that's running automatically on the CSS?

**Comment by meg-gutshall** [comment]
Hey Polly! From what I've seen so far this looks good! I'll fix up the CSS and push back those changes later tonight with a description of what I did. There's some utility classes built in so it's easier to use. 😄 

Thank you for your work! Even just putting the basic build up there and letting me fix up the CSS is helpful—or else I'd just be building the whole page on my own. I will get back to you soon!

**Review by meg-gutshall — APPROVED** [review]
Hey @pollygee! I made some changes so I can merge this into master. I'm going to leave this branch live so you have time to take a look at it. I made my commits pretty granular and added further description to some of them.

I'm also going to set up the pages we have left to build out with the front matter I _think_ we'll need for them but that could change. My goal behind building it out this way is to make it as easy as possible for Gia to add and edit content when she needs to.

**Comment by meg-gutshall** [comment]
As far as your original questions go...

<br>

> I don't know how to get the link to the anchor work from the exclusivity and equity section.

Me neither! That is something I can look into or if you decide to tackle it, let me know! 🙂 

<br>

> I don't know how to make the background change color all the way across the page over the sections that don't span the whole page and I don't know the color for that background stripe.

So this will change depending on whether you're building the page with multiple "blocks" as I've started calling them in my head (partials), or one partial for the page content. You can see in this commit 6e15fc1f80b14eb7dd360380d0ba1dd962f71a58 that I added the conditional `{% if block.bg %}{{ block.bg }}{% endif %}` line in the `section` class. This draws from the `about-us/landing-page.md` front matter and will include the class that gets passed in, if one exists. For a page that uses only one partial to hold its content, you'll use `page.bg` instead of `block.bg`.
In both cases, the `mixins/section_header.html` partial draws its background color from the `page.bg` front matter attribute. If the page has blocks, the `mixins/block_header.html` partial gets the background color from `block.bg`.
There are `background-color` variables set in the `default.css` file, but we've mainly been sticking to `grey-bg` and `white-bg`. Also in that file are the margin and padding classes you can see used throughout the code.

<br>

> I don't know if it's helpful, but I can try to keep building out as much as I know how to on some pages and have someone else do some of the styling? If that's not helpful, let me know what would be helpful.

I know jumping into a project is rough, but I'm super grateful for your help! 😍  I'm going to create a new branch tonight or tomorrow called mvp and just rip out a lot of the stuff we don't need right now. Hopefully that will make it easier for you to navigate the project. Please let me know if there's anything else I can do that will help you get more familiar with the code!

**Comment by pollygee** [comment]
Thanks @meg-gutshall for all the explanation.  I think I mostly understand what you said....though the new vocabulary is getting me a bit haha! But it's def all clearer than when I started.  I'll get it at some point.  thanks for fixing the rest of that up.  I felt bad it being in a partially completed state.  I'm glad it's better than nothing though!

I'm not confused with all the stuff that's in there, so don't rip anything out on my behalf.  I wonder if we could just comment pages out we're not going to show for MVP?  Whatever you think is best though 🤷

**Pull request merged**